mirrorcat/cmd/start.go

```diff
@@ -14,6 +14,7 @@ import (
 	"time"
 
 	"github.com/Azure/mirrorcat"
+	"github.com/go-redis/redis"
```

Comment:
@devigned, this lib uses BSD. Think that's okay?

Reply:
I don't think so, but you should put it through the LCA OSS tool anyway.

In fact, it would be great to build a go dep tool to automate submission of dependencies to LCA.

Reply:
Done. :) Also, I added a license flag to the version command which prints out all of the necessary attribution and things.